app/models/login.rb

```diff
@@ -33,9 +33,9 @@ class InvalidOAuth2Token < StandardError; end
     has_secure_password
   end
 
-  validates :identification, presence: true
-  validates :oauth2_token, presence: true
-  validates :single_use_oauth2_token, presence: true
+  validates :identification, presence: true, uniqueness: true
+  validates :oauth2_token, presence: true, uniqueness: true
+  validates :single_use_oauth2_token, presence: true, uniqueness: true
```

Comment:
`single_use_oauth2_token` is always generated right? so I couldn't figured how to test it